commands/livecmd.go

```diff
@@ -31,6 +33,10 @@ import (
 	"sigs.k8s.io/cli-utils/pkg/provider"
 )
 
+// resourceGroupEnv is an environment variable which hides code to implement
+// the ResourceGroup inventory object and migrating to ResourceGroup inventory.
+const resourceGroupEnv = "RESOURCE_GROUP_INVENTORY"
```

Comment:
This env variable is very specific to this functionality. The search-replace functionality will probably require something similar. I think we should consider adding something like an `ALPHA_FEATURES` flag that we could use for those as well.
@phanimarupaka

Reply:
I think we should keep them separate. You can imagine that someone would want one but not the other. And it's not very difficult to get them both -- just set two env vars.

Reply:
Let's also keep in mind, that this code should only be hidden behind this environment variable for a couple weeks at most (until the fishfood testing and feedback is completed); let's not overthink this.

Reply:
As features are not correlated, we can have one flag for each feature. It will be clear to the users as to what they are opting into. So better to have separate flags for each major feature. One per each area(e.g. cfg) at the least.